viscy/utils/meta_utils.py

```diff
@@ -127,6 +127,7 @@ def generate_normalization_metadata(
                 process=f"calculating channel statistics {channel}/{list(channel_ids)}",
             )
             position_statistics = dataset_statistics | {
+                "dataset_statistics": dataset_level_statistics,
```

Comment:
It should already exist. Can you test with a toy dataset without this line?

https://github.com/mehta-lab/VisCy/blob/529340db3d2750a0a5b2eb04a2a452df32e3d56f/viscy/utils/meta_utils.py#L112-L114

Reply:
I suspect that the dataset we looked at earlier was processed with an older version.

Reply:
See https://github.com/mehta-lab/VisCy/pull/68/commits/0a15f311453565fa6640c1e3deaa6d7ceadeb008 and https://github.com/mehta-lab/VisCy/pull/68/commits/c13400c0c951e9130ab70e9a255ff97f98676e68